src/components/Homepage/BackgroundImage.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from "@emotion/styled";
+
+interface IProps {
+  url: string;
+  backgroundSize?: boolean;
+}
+export const BackgroundImage = styled.div<IProps>`
+  background: url(${props => props.url}) no-repeat center;
+  background-size: ${({ backgroundSize }) => (backgroundSize ? "cover" : "")};
```

Comment:
have the `backgroundSize` prop as a boolean but rename it to something that indicates that it is a boolean, for example `isCoverBackground`
also if `false` set `auto`, not empty string